### ISSUE TITLE
How to execute a binary build with cppwinrt?

### ISSUE THREAD

**Issue by KHofer** [Microsoft/cppwinrt#161]
I've managed to check out the examples, modify them and run them in the visual studio debugger.
But when I am compiling a release build, I can not run the executable. I am getting this:
```
This application can only run in the context of an app container.
```
How can I deploy such a binary?

**Comment by kennykerr** [comment]
You need to create and install an app package containing your executable. Of course, a desktop/console app doesn't require this. 

Visual Studio can help you do this: Project > Store > Create App Package

This issue is not specific to C++/WinRT so you may better served asking related questions on [Stack Overflow](http://stackoverflow.com/).

**Comment by VSChawathe** [comment]
I did use [MakeAppx](https://docs.microsoft.com/en-us/windows/uwp/packaging/create-app-package-with-makeappx-tool) and [sideload](https://docs.microsoft.com/en-us/windows/uwp/packaging/packaging-uwp-apps#sideload-your-app-package) then just returned to using /ZW within Visual C++. I enjoyed code highlighting with KDevelop (think C++ only Visual Studio), set up to use MSBuild, but got tired before working out how to attach the debugger. /ZW only necessitates calling init_apartment () and disabling related warning if one finds it bothering. The trouble it saves from manual packaging is well worth.. besides the newer 2017 release Visual Studio's updates are improvising leaner C++ installs based on one's focus area.
